### PR TITLE
Patch sale_quotation_number for 9.0

### PR DESCRIPTION
**Pull request by whnr**
My first pull request :1st_place_medal: 
I patched the sale_quotation_number for 9.0, I tested it locally in my instance. Behaviour is as expected and explained. Maybe it's useful for someone.
––Flo

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
Hi @whnr 

Thanks a lot for your contribution! What you are doing it's actually migrating the module to v9 so please follow these [instructions](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0) and have a look to the [migration issue in this repository](https://github.com/OCA/sale-workflow/issues/223) for more information on the OCA standards and work status.

File: README.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ addon | version | summary
 [sale_procurement_group_by_line](sale_procurement_group_by_line/) | 9.0.1.0.0 | Base module for multiple procurement group by Sale order
 [sale_procurement_group_by_requested_date](sale_procurement_group_by_requested_date/) | 9.0.1.0.0 | Groups pickings based on requested date of order line
 [sale_product_set](sale_product_set/) | 9.0.1.0.1 | Sale product set
+[sale_quotation_number](sale_quotation_number/) | 9.0.1.0.0 | Different sequence for sale quotations
```
Comment (LoisRForgeFlow) [inline README.md:37]: There is no need to modify this file, it is automatically updated nightly.

**Comment by rafaelbn** [comment]
Hi @whnr , we are very happy to have you here bus as commented please follow https://github.com/OCA/maintainer-tools/wiki#migration

Thanks!

**Comment by rousseldenis** [comment]
Closing this as it is very old. Please feel free to reopen it if needed

**Pull request closed without merging**